packages/vulcan-forms/lib/components/FormSubmit.jsx

```diff
@@ -12,12 +14,35 @@ const FormSubmit = ({
   revertCallback,
   document,
   deleteDocument,
+  collection,
   collectionName,
   classes,
+  currentUser,
 }, {
   isChanged,
   clearForm,
-}) => (
+}) => {
+
+  // Delete permission check
+  let canDelete = false;
+
+  if (Users.isAdmin(currentUser)) {
```

Comment:
Seems overkill, don't we have a function to do this check already (Users.can)?

Reply:
You're right, i'll simplify this. Should I keed the admin's bypass ?

Reply:
I think it is already included in the existing Users functions (and it should be if it's not the case), so no you don't need to make it explicit normally

Reply:
You should check at the source code for the permissions in the users package, there are functions that should do the job in one line